packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/decode/DecodeForm.tsx

```diff
@@ -118,32 +121,55 @@ export function DecodeForm() {
       return;
     }
 
+    // Try to decode as resource
     try {
-      // Attempt to decode as table
       const resource = hexToResource(encodedData as Hex);
       if (resource) {
         setDecoded({ type: "resource", id: encodedData, resource });
+        return;
       }
-    } catch {
-      // an error here just means the encoded data is not a resource
+    } catch (error) {
+      console.error("Error decoding resource:", error);
```

Comment:
no strong opinion but feel like logging this error is not really needed since we're kind of expecting this to fail unless it's actually a resource id

Reply:
yeah I see the reasoning, removed `console.error`